qcfractal/interface/collections/dataset.py

```diff
@@ -301,64 +234,48 @@ def query(self,
 
         driver, keywords, program = self._default_parameters(driver, keywords, program)
 
-        if not reaction_results and (self.client is None):
+        if not local_results and (self.client is None):
             raise AttributeError("DataBase: FractalClient was not set.")
 
         query_keys = {
-            "method": method.lower(),
-            "basis": basis.lower(),
-            "driver": driver.lower(),
+            "method": method,
```

Comment:
Do you not need to cast these to lower anymore?

Reply:
Slowly pushing this into the server itself so that we only have to worry about it in one spot.